code/modules/reagents/newchem/drinks.dm

```diff
@@ -231,5 +233,6 @@
 	result = "synthignon"
 	required_reagents = list("synthanol" = 1, "wine" = 1)
 	result_amount = 2
+	mix_message = "The chemicals mix to create a fine, red substance."
```

Comment:
Fine like red wine. That's what I was thinking so if you have something better lay it on me